src/qae/object-properties.js

```diff
@@ -76,6 +76,11 @@ const properties = {
      */
     label: 'Totals',
   },
+  /**
+   * use pagination or continuous scroll. If undefined, defaults to using pagination
+   * @type {boolean}
```

Comment:
As this is documented. The property is not optional

Reply:
Sorry not following. Do you mean that it shouldn't be optional but it is now?

Reply:
Hmm you are right. I was thinking for a newly created chart this could be undefined, but then it willl be "filled in" by this definition. Then the second sentence becomes overkill to me